index.html

```diff
@@ -5,22 +5,182 @@
+        <div class="header-top">
+            <div class="wrapper">
+                <nav>
+                    <ul class="top-nav">
+                        <li><img src="images/icons/fb-icon.svg" alt="fb-icon"></li>
+                        <li><img src="images/icons/twitter-icon.svg" alt="twitter-icon"></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+
+        <div class="header-main">
+            <div class="wrapper">
+                <nav class="main-nav">
+                    <div class="icon-nav">
+                        <a href="index.html"><img src="images/logo.jpg" alt="logo image"></a>
+                    </div>
+                    <div class="icon-menu-mobile">
+                        <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+                    </div>
+                    <ul class="bottom-nav">
+                        <li><a  class="active" href="about.html">About</a>
+                        <li><a href="#program">Mission</a></li>
+                        <li><a href="#testimonials">Testimonials</a></li>
+                        <li><a href="#partners">Partners</a></li>
+                        <li><a href="#">Blog</a></li>
+                        <li><button type="button">Follow US</button></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+
+        <div class="nav-mobile">
+            <div class="cross">
+                <img src="images/icons/cross.svg" alt="cross">
+            </div>
+            <ul class="container">
+                <li><a href="index.html">Home</a>
+                <li><a href="about.html">About</a>
+                <li><a href="#program">Mission</a></li>
+                <li><a href="#testimonials">Testimonials</a></li>
+                <li><a href="#partners">Partners</a></li>
+                <li><a href="#">Blog</a></li>
+            </ul>    
+        </div>
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="wrapper headline-content">
+                <span>"Coding Con"</span>
+                <h1>LIVE CODING PROGRAM IN TREND 2023</h1>
+                <div class="headline-description">
+                    <p>Pair programming is an software development technique in which two programmers work together at one workstation. One, the driver, writes code while the other, the observer or navigator, reviews each line of code as it is typed in. The two programmers switch roles frequently.</p>
+                </div>
+                <div class="headline-footer">
+                    <h2>Upcoming Event - Unlimited Seats (28 ~ 31 March 2023)</h2>
+                    <h3>@USA - Cal Tech University</h3>
+                    <h3> <<<i> with @ahroniy </i>>></h3>
+                </div>
+            </div>
+        </section>
+
```

Comment:
As requested by the previous reviewers, it will be good if you stick to the given design template as closely as possible.

Kindly let:
- the design of the `LIVE CODING PROGRAM IN TREND 2023` in your project to look like the `CREATIVE COMMON GLOBAL SUBMIT 2015`
- the section mentioned above to have some background design as Cindy designed.
- the `Pair programming is an software development ...` text design and background to look like the `A joyful celebration ...` text design and background design (including width) to match Cindy's design. 😊

Reply:
✅ Edited as per your suggestion. Added A image background to the Main Heading Text and made adjustment according to the project description. 
thank you 😄